public/locales/en/common.json

```diff
@@ -190,12 +191,13 @@
     "RENAME": "Rename",
     "CENTER_RENAMED": "Center Renamed Successfully!",
     "SEND_REQUEST": "Send Request",
-    "SURE_DELETE_CENTER": "Are you sure you want to Delete Center?",
+    "SURE_DELETE_CENTER": "Are you sure you want to Delete ",
     "CENTER_UPDATE_SUCCESSFULLY": "Center has been Successfully Updated",
     "CENTER_DELETE_SUCCESSFULLY": "Center has been Successfully Deleted",
     "CENTER_CREATED_SUCCESSFULLY": "Center Created Successfully",
     "SEARCHBAR_PLACEHOLDER": "Search Center..",
-    "NOT_ABLE_CREATE_CENTER": "*Not able to create center for selected Data"
+    "NOT_ABLE_CREATE_CENTER": "Something went wrong",
```

Comment:
@upendraTekdi key and value should match

Reply:
done